subprojects/core/src/main/java/org/gradle/util/internal/BuildCommencedTimeProvider.java

```diff
@@ -19,7 +19,7 @@
 import org.gradle.internal.service.scopes.Scopes;
 import org.gradle.internal.service.scopes.ServiceScope;
 
-@ServiceScope(Scopes.BuildSession.class)
+@ServiceScope(Scopes.BuildTree.class)
```

Comment:
💭 The changes here and on `Factory` are incorrect scopes you discovered in the course of this work?

Reply:
Yes, exactly.